language-design.md

```diff
@@ -17,7 +17,81 @@ For each quote, describe what it was about the quote that led you pick it.
+> of those who work on it and use it. This tends to make users stay with it as
+> time goes by; they will take joy in working hard and helping out if they know
+> that their wants and needs have some weight and their hard work can change the
+> plan for the better [Steele, 1998]
+
+While flexibility as emphasized in this quote and throughout Steele's paper
+are foundational to good software development and general-purpose programming
+language development, we feel that a stricter approach when building out a DSL
+is essential in avoiding feature-bloat. Anthony pointed out an analagous duality
+of development styles prompted by this quote: Cathedral-Style is to Bazaar-Style
+as the [Waterfall Model](https://en.wikipedia.org/wiki/Waterfall_model) is to
+the [Agile Model](https://en.wikipedia.org/wiki/Agile_software_development).
+The former one, through a deep design and requirement definition process,
+produces a sort of master plan while the other is a more itereative,
+adaptive approach even once building (or coding) has started. While the Agile
+model can be extremley useful in more general software projects, for a DSL a
+so-called "master plan" can be extemely useful.
+
+While Ross mostly agrees with this analogy, he argues that the actual look and
+feel of the developing DSL should be up for debate and evolving&mdash;especially
+in its early days. Defining clear features and overall goals of what should be
+"easy, difficult, and impossible" is important, but the syntax decisions should
+be a more evolving component. As Steele mentioned in the quote above, the
+iterative process, if done with care, not only serves to better the final
+product, but it also serves to provide users "joy"!
+
+***Technical Wizards: The Dangers of Magic***
+
+> The problem is that programming languages are created either by committee or
+> by extreme technical wizards with magical math powers. The broad computer
+> science academic community has not paid a tremendous amount of attention to
+> programming language usability. [Pavlus, 2012]
+
+This quote embodies a fundamental issue with the devlopment of Programming
+Languages: many programming languages were designed without the many possible
+end-users who may use them. Instead, people build what they are comfortable with
+and assume others are also comfortable with it. A simple example of this is
+reading Racket. It may seem to make sense after learning about Lambda Calculus
+and the theory of Functional Programming; however, to a non-expert user it can
+be complicated, especially if you want to create usable *and* readable programs
+for others.
+
+Many times, it seems like programming languages are decorated with syntactic
+*aspartame*&mdash;it's not syntactic *sugar* since it is just a language creator
+adding unnatural, obfuscated, horn-tooting code (although some may argue it is
+meant to write code more succinctly).
+
+This dichotomy of committee-design versus technical wizard-design parallels the
```

Comment:
I think you're misinterpreting the quote - both the committees and the wizards have the same problems. The real dichotomy is between the wizard/committee side, and the bazaar side.